src/Annotation/ApiResource.php

```diff
@@ -256,6 +256,13 @@
      */
     private $sunset;
 
+    /**
+     * @see https://github.com/Haehnchen/idea-php-annotation-plugin/issues/112
+     *
+     * @var array
+     */
+    private $cacheHeaders;
```

Comment:
trully sorry but I forgot to mention that you also need to add ` @Attribute("cacheHeaders", type="array")` at the top of the file (in the alphabetical order) :disappointed:

Reply:
You had to comment on so many changes, bound to miss one - thanks for helping me improve my skills.